.github/workflows/ci.yml

```
@@ -26,4 +26,19 @@ jobs:
             bundle install --jobs 4 --retry 3
     
         - name: Run tests
-          run: bundle exec rake test
\ No newline at end of file
+          run: bundle exec rake test
+
+        - name: Verify schema is up-to-date
+          run: |
+            bundle exec rake graphql:update_schema
+            git diff --exit-code
+            exit $?
+
+        - name: Validate GraphQL queries
+          run: |
+            export EXITCODE=$(grep -E '^(mutation|query) [a-z]' lib/queries.graphql || echo $?)
+            if [[ $EXITCODE -eq 0 ]]; then
+              echo "Failed to validate GraphQL queries."
+              echo "Ensure that GraphQL named queries are in title-case."
```

```suggestion
              echo "Ensure that GraphQL named queries are in TitleCase"
```

Generally I've seen `camelCase`, `snake_case`, `kebab-case`, and `TitleCase` written in their respective casing.